backend/apis.js

```diff
@@ -0,0 +1,129 @@
+"use strict";
+Object.defineProperty(exports, "__esModule", { value: true });
+const { Client } = require('pg');
+const client = new Client({
+    user: "postgres",
+    password: "test123",
+    host: "localhost",
+    port: 5432,
+    database: 'userdb'
+});
+let express = require("express");
+let app = express();
+let fs = require('fs');
+let bodyParser = require('body-parser');
+app.listen(3000);
+app.use(bodyParser.json());
+function getData(req, res) {
+    client
+        .connect()
+        .then(() => console.log("Successfully connected"))
+        .catch(e => console.log(e));
+    client.query('select users.empid,users.firstname,users.middlename,users.lastname,users.email,users.phoneno,roles.fname,users.address,customer.website from users,customer,roles where users.role=roles.fkey AND users.customer_id=customer.customer_id order by users.empid asc;', (err, result) => {
+        console.log(result.rows);
+        res.status(200).json(result.rows);
+    });
+}
+app.get("/class2", getData);
+app.delete("/delete/:id", function (req, res) {
+    let id1 = req.params.id;
+    client.query(`delete from users where empid=${id1};`, (err, result) => {
+        res.send();
+    });
+});
+app.put('/update/:id', function (req, res) {
+    const value = req.body;
+    const id1 = req.params.id;
+    client.query(`update users set firstname=$1,middlename=$2,lastname=$3,email=$4,phoneno=$5,role=$6,address=$7,customer_id=$8
+  where empid=${id1};`, [value.firstname, value.middlename, value.lastname, value.email, value.phoneno, value.fname, value.address, value.customer_id]);
+    res.send();
+    console.log(value);
+});
+app.post('/addrow', function (req, res) {
```

Comment:
apis  were related to users. So, API URLs should have been /addUser, /deleteUser